superset-frontend/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -213,10 +213,13 @@ class DatasourceControl extends React.PureComponent {
       </Menu>
     );
 
+    // eslint-disable-next-line camelcase
+    const { health_check_message: healthCheckMessage } = datasource;
+
     return (
       <Styles className="DatasourceControl">
         <ControlHeader {...this.props} />
-        <div>
+        <div style={{ display: 'flex' }}>
```

Comment:
instead of an inline style, should we use a `styled.div` to apply this style?

Reply:
fixed.